packages/core/src/libraries/hook/utils.test.ts

```diff
@@ -16,6 +16,13 @@ mockEsm('#src/utils/sign.js', () => ({
   sign: () => mockSignature,
 }));
 
+const mockNanoId = 'mockNanoidId';
+await mockEsmWithActual('@logto/shared', () => ({
+  // eslint-disable-next-line unicorn/consistent-function-scoping
```

Comment:
nit: try `jest.fn`? maybe the lint error will disappear

Reply:
Good call, updated.